### PR TITLE
CRM-19134 - Missing French overseas departments.

### PR DESCRIPTION
**Pull request by ggrzywna**
Missing French overseas departments.
Refs: https://issues.civicrm.org/jira/browse/CRM-19134

### CODE REVIEW

File: CRM/Upgrade/Incremental/sql/4.7.10.mysql.tpl [hunk 1]
```diff
@@ -21,3 +21,11 @@ INSERT INTO
    `civicrm_option_value` (`option_group_id`, {localize field='label'}label{/localize}, `value`, `name`, `grouping`, `filter`, `is_default`, `weight`, `is_optgroup`, `is_reserved`, `is_active`)
 VALUES
    (@option_group_id_ext, {localize}'{ts escape="sql"}odt{/ts}'{/localize}, @option_group_id_ext_val+1, 'odt', NULL, 0, 0, @option_group_id_ext_wt+1, 0, 1, 1);
+
```
Comment (seamuslee001) [inline CRM/Upgrade/Incremental/sql/4.7.10.mysql.tpl]: Note as 4.7.10 is in RC this will need to be moved to 4.7.11 file

**Comment by totten** [comment]
jenkins, ok to test

**Comment by totten** [comment]
Ooh, you patched everything (the `.tpl` *and* the `civicrm_generated.mysql` and the upgrader). Good call. :)

Aside: The "NFC" label is a bit subjective. In this case, it feels more functional to me (e.g. it actually changes the functionality as experienced by users and requires executing some code during upgrade). However, with that said, it is a pretty conservative/low-risk area.

**Comment by seamuslee001** [comment]
I have tested this on a CiviBuild enviornment, first by getting CiviBuild to reinstall it following adding patch and then after CiviBuild upgrading, I can confirm that the new french departments do then exist in both circumstances

**Comment by monishdeb** [comment]
Yup I agree with @seamuslee001 it works on both upgrade and install. Merging ..

**Pull request merged**